plugins/container/src/caps/async/async.tpp

```diff
@@ -48,9 +48,6 @@ void generate_async_event(const char *json, bool added, bool initial_state)
     }
     else
     {
-        // set ts = now + 1s to leave some space for enqueued syscalls to be
-        // enriched
-        enc.set_ts(get_current_time_ns(1));
```

Comment:
We don't need this trick anymore, since we will now send the `removed` event after the final `procexit` in the container.